topo/node/srl/srl.go

```diff
@@ -0,0 +1,111 @@
+package srl
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	expect "github.com/google/goexpect"
+	topopb "github.com/google/kne/proto/topo"
+	"github.com/google/kne/topo/node"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+)
+
+func New(pb *topopb.Node) (node.Implementation, error) {
+	cfg := defaults(pb)
+	proto.Merge(cfg, pb)
+	node.FixServices(cfg)
+	return &Node{
+		pb: cfg,
+	}, nil
+}
+
+type Node struct {
+	pb *topopb.Node
+}
+
+func (n *Node) Proto() *topopb.Node {
+	return n.pb
+}
+
+var (
+	spawner = defaultSpawner
```

Comment:
you don't need this unless you are using expect for anything

Reply:
ah, yes, sure, I will add it back when I will have config/tls done
removed in 20baf3b